CONTRIBUTING.md

```diff
@@ -37,6 +37,8 @@ To make a feature request, document the problem and a sketch of the solution wit
 
 Your goal will be to convince others that your proposal is a useful addition to the language and recruit TC39 members to help turn your request into a proposal and shepherd it into the language. Once a proposal is introduced to the committee, new features are considered by the committee according to the [TC39 process document](https://tc39.es/process-document/).
 
+Proposals should be backed by a repository created from the [TC39 Template for Proposals](https://github.com/tc39/template-for-proposals). Once adopted to Stage 1 by the committee, the proposal repository should be transferred into the tc39 org on GitHub.
```

Comment:
I'm not sure we have consensus that proposals "should" be created from the template; that's there more as a convenient option.

Reply:
I changed "should" to "may"